packages/core/package.json

```diff
@@ -36,33 +35,18 @@
     "backbone-undo": "0.2.6",
     "codemirror": "5.63.0",
     "codemirror-formatting": "1.0.0",
+    "html-entities": "~1.4.0",
     "promise-polyfill": "8.1.3",
     "underscore": "1.13.1"
   },
   "devDependencies": {
-    "@babel/cli": "7.24.8",
-    "@babel/preset-typescript": "7.24.7",
-    "@types/jest": "29.5.12",
     "@types/markdown-it": "14.1.2",
-    "@typescript-eslint/eslint-plugin": "8.2.0",
-    "@typescript-eslint/parser": "8.0.1",
-    "@vuepress/plugin-google-analytics": "1.8.2",
-    "@vuepress/types": "1.9.10",
-    "documentation": "14.0.3",
-    "eslint": "8.57.0",
-    "eslint-config-prettier": "9.1.0",
-    "eslint-plugin-jest": "28.8.0",
-    "eslint-plugin-prettier": "5.1.3",
     "grapesjs-cli": "4.1.3",
```

Comment:
There is a conflict in the repo with `webpack`, as vuepress relies on a different version of webpack as to `grapesjs-cli`, we can resolve this better and share webpack when we have the cli in the monorepo.